pages/index.page.js

```diff
@@ -214,7 +214,7 @@ export default function Home() {
             <div className="left-content3">
               <div className="section-tittle section-tittle2 mb-20">
                 <div className="front-text">
-                  <h2>Why to be Mentor at Grabtern?</h2>
```

Comment:
please revert this! I'll merge then

Reply:
> please revert this! I'll merge then
@anmode 
Sure thing ! Just reverted it . Let me know if this works .